src/Windows/Avalonia.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -1496,6 +1496,86 @@ internal static int WaitForMultipleObjectsEx(int nCount, IntPtr[] pHandles, bool
         [DllImport("user32.dll")]
         internal static extern int SetWindowCompositionAttribute(IntPtr hwnd, ref WindowCompositionAttributeData data);
 
+        [DllImport("Imm32.dll", SetLastError = true)]
```

Comment:
pendantic nit: make the dll name lowercase

Reply:
roger roger